### ISSUE TITLE
Completed torrents Queue number 0

### ISSUE THREAD

**Issue by AlexR00T** [tympanix/Electorrent#206]
**Client:**
- [ ] µTorrent
- [x] qBittorrent
- [ ] Transmission
- [ ] rTorrent
- [ ] Deluge

**Client Version:**
4.3.1
**Operating System:**
MacOS 11
**Application Version:**
2.7.2.462
**Description:**
This new version brought to Queue 0 in front of torrents. I loved how it worked before without numbers in front of completed torrents. I simply had all torrents sorted by Queue and then once completed they went at bottom of it.
![Screen Shot 2020-12-26 at 09 02 46](https://user-images.githubusercontent.com/16637549/103147840-2b5b5780-4759-11eb-8d6f-337a9e4e111a.jpg)

**Comment by tympanix** [comment]
Good catch. This bug was introduced in 78312b2d2c75dd660c7bdc7bf81b1815502044df as a fix for Transmission, because Transmission starts it queue at zero. Fix should apply another fix which does not affect other client implementations.

**Comment by tympanix** [comment]
Fixed in #573
